qemu/tests/rng_hotplug.py

```diff
@@ -38,32 +103,17 @@ def run(test, params, env):
             vm.devices.set_dirty()
             new_dev = qdevices.QDevice("virtio-rng-pci",
                                        {'id': 'virtio-rng-pci-%d' % num})
+            hotplug_rng(vm, new_dev)
             dev_list.append(new_dev)
-            error_context.context("Hotplug %s" % new_dev, logging.info)
-            output = new_dev.hotplug(vm.monitor)
-            time.sleep(2)
 
-            error_context.context("Check %sfrom qtree after hotplug" % new_dev,
+        # Run test after hotplug
+        if test_after_hotplug and i == xrange(repeat_times):
```

Comment:
should be "i == repeat_times-1"?
\>>> repeat_times = 10
\>>> xrange(repeat_times)
xrange(10)
\>>> type(xrange(repeat_times))
\<type 'xrange'\>